src/blocks/menu-food/scss/style.scss

```diff
@@ -158,3 +157,31 @@
 .nova-food-menu__section {
 	break-inside: avoid;
 }
+
+.is-style-basic {
+	.nova-food-menu-item {
+		grid-template-areas:
```

Comment:
I guess this could be
```css
grid-template-areas:
			'label'
			'title'
			'prices'
			'description';
```
and there would be no overwrites for the highlighted state.

Reply:
Thank you for the suggestions.